src/App.js

```diff
@@ -4,46 +4,48 @@ import { Button, Link } from "@mui/material";
 import NavBar from "./components/NavBar";
 import HomePage from "./pages/HomePage";
 import {
+  BrowserRouter,
   BrowserRouter as Router,
-  Switch,
   Route,
   Routes,
 } from "react-router-dom";
+import theme from "./components/theme";
+import { ThemeProvider } from "@mui/system";
+import PortfolioPage from "./pages/PortfolioPage";
 import BlogPage from "./pages/BlogPage";
+import ResumePage from "./pages/ResumePage";
 
 function App() {
   return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-        <Button variant="contained"> Hello world!</Button>
-        <NavBar></NavBar>
-        {/* <HomePage></HomePage> */}
-        {/* <BlogPage /> */}
+    <ThemeProvider theme={theme}>
+      <BrowserRouter>
+        <div className="App">
```

Comment:
make this a component

Reply:
What do you mean by making this a component?